ReactWindows/ReactNative/Views/Text/ReactTextShadowNode.cs

```diff
@@ -117,7 +122,15 @@ public override void OnCollectExtraUpdates(UIViewOperationQueue uiViewOperationQ
 
             if (_inline != null)
             {
-                uiViewOperationQueue.EnqueueUpdateExtraData(ReactTag, _inline);
+                var args = new Tuple<Inline, TextAlignment?, double?, int?, int?>(
```

Comment:
You can use Tuple.Create with generic type inference. You'll still have to do the 'as' check in the view manager with the full type, but at least it is only one spot to maintain the type.  Up to you on which is clearer.